apps/cloudfront-functions/src/viewer-request-handler.ts

```diff
@@ -0,0 +1,27 @@
+// This code is executed in the CloudFront Functions JavaScript runtime. In this
+// context we prefer performance over immutability.
+// https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/writing-function-code.html#function-code-modify-request
+
+/* eslint-disable functional/immutable-data */
+/* eslint-disable functional/no-expression-statements */
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const handler = (
```

Comment:
is there any reason on why we cannot export this?

Reply:
There are some [restrictions on Cloudfront Functions](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/functions-javascript-runtime-features.html). One of them is the `export` keyword, [it is not allowed](https://www.uglydirtylittlestrawberry.co.uk/posts/unit-testing-cloudfront-functions/#:~:text=Because%20there%20are%20no%20exports%20allowed.) 😢